tests/fixture_plugins/demultiplex_fixtures/flow_cell_fixtures.py

```diff
@@ -0,0 +1,172 @@
+"""Fixtures for flow cell objects."""
+from pathlib import Path
+
+import pytest
+
+from cg.constants.demultiplexing import BclConverter, DemultiplexingDirsAndFiles
+from cg.models.flow_cell.flow_cell import FlowCellDirectoryData
+
+# Functional flow cells
+
+
+@pytest.fixture(scope="module")
+def hiseq_x_single_index_flow_cell(
+    hiseq_x_single_index_flow_cell_dir: Path,
+) -> FlowCellDirectoryData:
+    """Return a single-index HiSeqX flow cell."""
+    return FlowCellDirectoryData(flow_cell_path=hiseq_x_single_index_flow_cell_dir)
+
+
+@pytest.fixture(scope="module")
+def hiseq_x_dual_index_flow_cell(
+    hiseq_x_dual_index_flow_cell_dir: Path,
+) -> FlowCellDirectoryData:
+    """Return a dual-index HiSeqX flow cell."""
+    return FlowCellDirectoryData(flow_cell_path=hiseq_x_dual_index_flow_cell_dir)
+
+
+@pytest.fixture(scope="module")
+def hiseq_2500_dual_index_flow_cell(
+    hiseq_2500_dual_index_flow_cell_dir: Path,
+) -> FlowCellDirectoryData:
+    """Return a dual-index HiSeq2500 flow cell."""
+    return FlowCellDirectoryData(flow_cell_path=hiseq_2500_dual_index_flow_cell_dir)
+
+
+@pytest.fixture(scope="module")
+def hiseq_2500_custom_index_flow_cell(
+    hiseq_2500_custom_index_flow_cell_dir: Path,
+) -> FlowCellDirectoryData:
+    """Return a custom-index HiSeq2500 flow cell."""
+    return FlowCellDirectoryData(flow_cell_path=hiseq_2500_custom_index_flow_cell_dir)
+
+
+@pytest.fixture()
+def novaseq_6000_post_1_5_kits_flow_cell_data(flow_cells_dir: Path) -> FlowCellDirectoryData:
+    return FlowCellDirectoryData(Path(flow_cells_dir, "230912_A00187_1009_AHK33MDRX3"))
+
+
+@pytest.fixture()
+def novaseq_6000_pre_1_5_kits_flow_cell_data(flow_cells_dir: Path) -> FlowCellDirectoryData:
+    return FlowCellDirectoryData(Path(flow_cells_dir, "190927_A00689_0069_BHLYWYDSXX"))
+
+
+@pytest.fixture()
+def novaseq_x_flow_cell_data(flow_cells_dir: Path) -> FlowCellDirectoryData:
+    return FlowCellDirectoryData(Path(flow_cells_dir, "20231108_LH00188_0028_B22F52TLT3"))
+
+
+# Broken flow cells
```

Comment:
maybe the docstring can contain why a flow cell is broken within the docstring or the actual fixute name so we can better distinguish them?

Reply:
yes, that is the purpose of my next PR. My idea is that many of these fixtures can be removed or replaced by existing ones. The fixtures for functional flow cells should be the ones above and no more, because those are the flow cells that are well-documented.